components/MenuContent/MenuContent.tsx

```diff
@@ -89,71 +97,112 @@ const MenuContent: FC<MenuContentProps> = ({ setDrawerOpen }) => {
   };
 
   return (
-    <Box display="flex" flexDirection="column" sx={{ alignItems: { xs: 'center', md: 'flex-start' } }} minHeight="80vh">
+    <Box
+      display="flex"
+      flexDirection="column"
+      sx={{ alignItems: { xs: 'center', md: 'flex-start' } }}
+      minHeight="60vh"
+      style={{
+        backgroundColor: 'rgba(255,255,255,1)',
+        // backgroundImage: 'linear-gradient(to top left, #d70a84, #51127f)',
+        boxShadow: '0 4px 6px -1px #0000001a,0 2px 4px -1px #0000000f',
+        height: '70vh',
+        width: '14vw',
+        borderRadius: '15px',
+        overflow: 'hidden',
+        margin: 0,
+        padding: 0,
+        justifyContent: 'center',
+        alignItems: 'flex-start',
+      }}
```

Comment:
Create separate styled component for that. Inlines style like display="flex" are ok bu t not that long :D